### PR TITLE
fix: attempt to fix cross client tests with new core API

### PR DESCRIPTION
**Pull request by harbu**
Need to pass id to createStream explicitly

### CODE REVIEW

**Review by teogeb — APPROVED** [review]

File: packages/cross-client-testing/src/main/java/com/streamr/client_testing/StreamTester.java [hunk 1]
```diff
@@ -50,13 +53,18 @@
...
                 SigningOptions.getDefault(), EncryptionOptions.getDefault(), websocketApiUrl, restApiUrl);
         options.setStorageNodeAddress(DEV_STORAGE_NODE_ADDRESS);
         options.setStorageNodeUrl(DEV_STORAGE_NODE_URL);
         this.creator = new StreamrClient(options);
         try {
-            stream = creator.createStream(new Stream(streamName, ""));
+            // TODO: consider cleaning below when 3.0.0 is landed (and the private static helper method)
+            String address = publicKeyFromPrivateKey(privateKey);
+            Stream protoStream = new Stream(streamName, "");
+            protoStream.setId(address + "/" + System.nanoTime());
```
Comment (timoxley) [inline packages/cross-client-testing/src/main/java/com/streamr/client_testing/StreamTester.java:66]: is it possible these execute in same nanosecond

**Review by timoxley — APPROVED** [review]
nice one, lgtm

File: packages/cross-client-testing/src/main/java/com/streamr/client_testing/StreamTester.java (hunk 1 above)
Comment (harbu) [reply packages/cross-client-testing/src/main/java/com/streamr/client_testing/StreamTester.java]: I read a bit about these and basically https://stackoverflow.com/questions/56809879/can-i-use-nanotime-instead-of-randomuuid:

> In my experience with the particular Java implementations I have used, the origin is the moment when the JVM starts up. This means I will most certainly see the same numbers all over again after the next JVM restart. 

So you have an already plausible chance to hit the same numbers when restarting JVM and running this test multiple times.

**Changed to use UUID.**

Comment (harbu) [reply packages/cross-client-testing/src/main/java/com/streamr/client_testing/StreamTester.java]: Ah it actually generates a random private key for which the public key is used as the first part of the stream id. So the above argument doesn't really apply. Well, I still feel better having it as UUID than nanoTime so will keep it as is.

**Pull request merged**